### PR TITLE
Publish swag-crowdsec mod

### PR DESCRIPTION
**Pull request by TheSpad**
Do not merge yet - pending v1.0 final release of the CrowdSec nginx bouncer.

### CODE REVIEW

**Comment by NemesisRE** [comment]
@TheSpad TheSpad v0.7.0 is marked as stable why wait for v1.0.0?

**Comment by TheSpad** [comment]
> @TheSpad v0.7.0 is marked as stable why wait for v1.0.0?

Because v1.0.0 makes substantial changes to the makeup and dependencies for the bouncer and releasing the mod with v0.7.0 would create substantial work to then migrate existing installs to v1.0.0.

**Comment by TheSpad** [comment]
v.1.0.0 is now fully released. Tested locally as thorougly as I can and seems to work as expected.

File: root/etc/cont-init.d/98-crowdsec [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+apk add -U --upgrade --no-cache \
+    gettext \
+    lua5.1 \
+    lua5.1-cjson \
+    lua-resty-http \
+    lua-sec \
+    nginx-mod-http-lua
+
+# Download nginx bouncer
+CROWDSEC_VERSION=$(curl -sX GET "https://api.github.com/repos/crowdsecurity/cs-nginx-bouncer/releases/latest" | awk '/tag_name/{print $4;exit}' FS='[""]');
```
Comment (Roxedus) [inline root/etc/cont-init.d/98-crowdsec:26]: I think having the option to enforce a url would be beneficial here

Comment (TheSpad) [reply root/etc/cont-init.d/98-crowdsec:26]: For applying a fixed version you mean?

Comment (Roxedus) [reply root/etc/cont-init.d/98-crowdsec:26]: yea

Comment (TheSpad) [reply root/etc/cont-init.d/98-crowdsec]: Done.

**Review by aptalca — APPROVED** [review]

**Pull request merged**